### DISCUSSION TITLE
cluster support

### DISCUSSION THREAD

**Discussion by aerdnach** [OptimalBits/bull#2383] [Q&A]
Hi everyone 👋!

From quick guide, in **cluster support** section:

NOTE: From version 3.2.0 and above it is recommended to use threaded processors instead. 

Is there any example? Thanks

**Comment by manast** [comment]
https://github.com/optimalbits/bull#separate-processes

**Reply by aerdnach** [reply]
Ok, thanks, was simply that :) 

Can I ask why form that version above it is a recommended approach?
